pkg/permissionclaim/permissionclaim_labeler.go

```diff
@@ -86,5 +94,23 @@ func (l *Labeler) LabelsFor(ctx context.Context, cluster logicalcluster.Name, gr
 		}
 	}
 
+	// for APIBindings we have to set the constant label value on itself to make the APIBinding
+	// pointing to an APIExport visible to the owner of the export, independently of the permission claim
+	// acceptance of the binding.
+	if groupResource.Group == apis.GroupName && groupResource.Resource == "apibindings" {
```

Comment:
why is this part of the this pkg ? shouldn't it be added to a controller that reconciles only bindings?

Reply:
disregard. I think it is correct. @ncdc can you confirm?

Reply:
I think it's fine here. If we decide to move it later, we can.